auth/localkeys.go

```diff
@@ -308,12 +309,15 @@ func ClearKeys() {
 func WaitForDelegateKeys(cancel <-chan interface{}) <-chan struct{} {
 	ch := make(chan struct{})
 	go func() {
-		for delegateKeys.localPrivate == nil || delegateKeys.masterPublic == nil {
-			select {
-			case <-dKeyCond.Wait():
```

Comment:
NewChannelCond object is often used in localkeys.go and token.go files and for me, it's hard to say in what exact code place we're not releasing this object for being able to close the channel and started establishing the RPC connections in the controller. Because of that I just simplified the logic. Now the goroutine only checking the existing keys every second until they wouldn't be load and then close the channel.